lib/web_ui/lib/src/engine/compositor/canvas.dart

```diff
@@ -97,8 +97,8 @@ class SkCanvas {
   }
 
   void drawColor(ui.Color color, ui.BlendMode blendMode) {
-    skCanvas.callMethod('drawColor', <dynamic>[
-      color.value,
+    skCanvas.callMethod('_drawColor', <dynamic>[
```

Comment:
This is a temporary work around because drawColor would free the Malloc'd float array, right? I think this is a strong signal that CanvasKit should not do that; I can make such changes.

Reply:
Yes, that's right. I added some comments clarifying why we're using the private methods here.

I suggest changing the behavior of the methods that automatically free the arrays to check if the arrays were already malloc'ed and not free them in that case.

Reply:
Changed to `drawColorInt`